source/concepts/environments/problem-and-solution.html.md.erb

```diff
@@ -30,120 +30,120 @@ For example, if we had an account for `fictional-business-unit-production` and s
 
 Since Data Transfer isn't taggable, you can't easily trace it back to the originating S3 bucket.
 
-### Issue 4: cloud waste
+### Issue 4: Cloud waste
 
-When holding all applications in a SDLC account, it becomes difficult to retire application infrastructure, even if they're covered by a good tagging policy. With cloud infrastructure, everything should be considered ephemeral and everything should be rebuildable from code.
+When holding all applications in a SDLC account, it becomes difficult to retire application infrastructure, even if they're covered by a good tagging policy. When deploying to the cloud, all infrastructure should be considered ephemeral and everything should be rebuildable from code.
 
-If you destroy the wrong infrastructure when retiring an application, it becomes costly to recreate. If you miss the deprovisioning of some infrastructure, you create cloud waste which is also costly.
+If you destroy the wrong infrastructure when retiring an application, it becomes costly to recreate. If you accidently miss some infrastructure during de-provisioning, you inadvertently create ongoing cloud waste which is also costly.
 
-Unless you've retired all applications within a shared AWS account, you can't close it, so there's always a chance there are some resources unaccounted for after an application has been refactored, replaced, or retired.
+Until you've retired all applications within a shared AWS account, you can't close it, so there's always a chance there are some resources left unaccounted for after an application has been refactored, replaced, or retired.
 
-#### Example of cloud waste
+#### Example: Cloud waste from uncertainty
 
 `example-a` has been retired, and is no longer of use. There are untagged resources that are thought to be part of `example-a`, but `team-a` is unsure, so they just leave it.
 
 #### Example of destroying the wrong infrastructure
 
-See [An extreme misclicking example](#an-extreme-misclicking-example).
+See [Example: Human error (misclick)](#example-human-error-misclick).
 
 ## What we investigated
 
-### Let teams decide
+### Option A: Let teams decide
 
-In the Modernisation Platform, we want to empower teams and give them the autonomy to do whatever they need to support their application and environments. One of the issues with delegating how their infrastructure is separated is that it:
+In the Modernisation Platform, we want to empower teams and grant them the autonomy to do whatever they need to support their applications and environments. Some complexities that can arise from delegating how infrastructure is separated are:
 
-- quickly becomes hard to track
-- can become messy
-- doesn't support the central alignment we're trying to achieve
 - people tend to go for the easiest option, which creates technical debt further down the line
+- infrastructure quickly becomes hard to track
+- it can become tangled and messy
+- doesn't support the central alignment we're trying to achieve 
```

Comment:
I'd prefer to reword this, as it's not clear to me exactly what this means

Reply:
If you feel this needs to be reworded you could send out an example to the whole team to see if anyone has complaints/suggestions.

Reply:
Personally? This is an early way of describing what P&A exists to do through hosting teams.